### DISCUSSION TITLE
How to change the text from English to any other language in play.html page?

### DISCUSSION THREAD

**Discussion by yashtandon113** [ant-media/Ant-Media-Server#4880] [FAQs] [answered]
When I try to change the text on player that reads "Your stream will start playing automatically when it is live" to another language, I get the text shown below.

**ã‚¹ãƒˆãƒªãƒ¼ãƒ ã¯ãƒ©ã‚¤ãƒ–**

Is there a way to make that happen?

**Comment by lastpeony** [comment, marked as answer]
You should enable UTF-8 encoding on the html page for to achieve this. Please add below lines on top of play.html page.

`<%@page pageEncoding="UTF-8" contentType="text/html; charset=UTF-8"%>`

After saving the changes, you will be able to change the text to other language.

Thank you!
